draft-ietf-sedate-datetime-extended.md

```diff
@@ -522,21 +520,23 @@ timestamps are inconsistent:
 
 As per {{Section 4.3 of RFC3339}} as updated by {{update}}, IXDTF
 timestamps may also forego indicating local time information in their
-{{RFC3339}} part.
+{{RFC3339}} part by using `Z` instead of a numeric time zone offset.
 The IXDTF timestamps in {{example-consistent}} (which represent the same
 instant in time as the strings in {{example-inconsistent}}) are not
 inconsistent because they do not assert any particular local time nor
 local offset in their {{RFC3339}} part.
-Instead, applications that receive these strings can base their
-local offset and local time calculations on the time zone suffix
-given, i.e., using the Europe/London time zone rules.
+Instead, applications that receive these strings can calculate the
+local offset and local time using the rules of the time zone suffix,
+such as Europe/London in the example below.
 
     2022-07-08T00:14:07Z[!Europe/London]
     2022-07-08T00:14:07Z[Europe/London]
-    2022-07-08T00:14:07-00:00[!Europe/London]
-    2022-07-08T00:14:07-00:00[Europe/London]
 {: #example-consistent title="No inconsistency in IXDTF timestamps"}
 
+Note that `-00:00` may be used instead of `Z`, because they have the
+same meaning according to {{update}}, but `-00:00` is incompatible
```

Comment:
We are using very specific wording elsewhere, I think we should do that here, too (and reference `{{ISO8601-2000}}`).

Reply:
OK, I replaced "ISO-8601" with "{{ISO8601-2000}}".  Was there another change you wanted to see to this text here?

Reply:
I replaced "incompatible with" with "not allowed by".